pyproject.toml

```diff
@@ -0,0 +1,31 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools.packages.find]
+where = ["."]
+# include = ["scout"]
+namespaces = false
+
+[project]
+name = "scout"
+readme = "README.md"
+version = "0.9"
+authors = [
+  {email = "chioke.harris@nrel.gov"},
+  {name = "Chioke Harris"},
+  {email = "jared.langevin@lbl.gov"},
+  {name = "Jared Langevin"}
+]
+requires-python = ">=3.8"
+license = {text = "Apache-2.0"}
+dependencies = [
+    "pandas >= 2.0",
```

Comment:
I'm not sure if we have a requirement for v2.2 versus v2.0, but v2.2 was the previously stated version requirement.

Reply:
The most recent pandas release was v2.1.1 though...

Reply:
In that case...v2.0 is good.